.github/workflows/publish-nightly-artifacts.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish nightly artifacts
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
You might consider triggering on a GitHub release instead of `main`. That way you're intentional about the release and it forces you to have those generated on the GitHub side too... see [this](https://github.com/1904labs/dom-to-image-more/blob/main/.github/workflows/publish.yml) for an example of how to setup the publish when a new [release is created](https://github.com/1904labs/dom-to-image-more/releases)

Reply:
Thanks, Marc.  I plan to use this workflow to push a nightly or "committly" docker build to a separate docker repo.  This way, we would have a stable "official" release triggered off tags. And we would create new artifacts off `main` for easier bleeding edge testing.  

What do you think?

Reply:
For example with Docker. 
`main` goes to `iridiumidentity/iridium-core-server-nightly`
tags go to `iridiumidentity/iridium-core-server`